charts/moondog-navigator/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 0.0.5
+appVersion: 0.0.6
```

Comment:
Not sure on this. I know @jwietelmann has an opinion on `appVersion` vs `version` since usually the `appVersion` denotes the version of the application being installed by the chart, where `version` is the chart version.

Reply:
Oh, yeah... I've definitely been confused by that while reading up on other k8s apps

Reply:
Since navigator the app is still on 0.0.1 (and not yet public) I'm going to merge and let this discussion inform the _future_ process 😈